exporter/datadogexporter/examples/collector.yaml

```diff
@@ -372,7 +372,7 @@ exporters:
       ##
       ## To attach tags to telemetry signals regardless of the host, use a processor instead.
       #
-      # tags: []
+      # tags: ["foo:bar", "baz:qux"]
```

Comment:
```suggestion
      # tags: ["team:infra", "<TAG_KEY>:<TAG_VALUE>"]
```

Reply:
What do you think of this instead ?

Reply:
The configuration is meant to show the default value, which is an empty list. Maybe we can add this example on the description above this?

Reply:
After discussing with @mackjmr offline I agree with the suggestion :)

Reply:
Sounds good to me!